src/app/components/table/VirtualTable.jsx

```diff
@@ -417,7 +417,7 @@ export default class VirtualTable extends PureComponent {
     columnIdx === 0 || f.get("visible", this.props.columns[columnIdx]);
 
   getVisibleElement = (elements, idx) =>
-    elements[this.visibleColumnIndices[idx]];
+    elements[this.props.visibleColumnOrdering[idx]];
```

Comment:
Was passiert an der Stelle mit dem vorherigen Feld `this.visibleColumnIndices`? WIrd das nicht mehr genutzt? Wird es an anderen Stelle noch genutzt? Falls es wo anders noch genutzt wird, ist das korrekt, oder sollte dort auch die Prop genutzt und das Feld dann entfernt werden?